app/views/layouts/mazer.html.erb

```diff
@@ -27,12 +27,17 @@
   </head>
 
   <body>
+    <!-- Only for connexion and reset password not display sidebar -->
+  <% unless current_page?('/connexion') || current_page?('/reset_password') %>
+      <%= render "layouts/sidebar" %>
```

Comment:
We should probably have a different layout all together for the session stuff. Like a session layout `session.html.erb` that is going to be specific for the password/login/etc